### ISSUE TITLE
Skip pin input

### ISSUE THREAD

**Issue by TechnoFlawgic** [Fredrum/chiaki#23]
So, I've got this working on my RPi 4 with RetroPie pretty well, but the one thing I can't get around is the pin entry after waking my console. I currently either have to turn on the console and login before using Chiaki, or have a keyboard plugged in to input the pin. Is there an entry I can put in the config file to save my pin, or any other work around that I'm not finding?

**Comment by Fredrum** [comment]
Hi sorry for late reply. You should only need to add the PIN once, when first registering, so I'm a bit confused about what you are seeing.
Once you have your config file in ~/.config/Chiaki/    you should not need to re-register anything.

**Comment by TechnoFlawgic** [comment]
So, I should have clarified better before: this isn't for the registration pin, but rather the security pin that I've set so that my kids don't log into my account. I'm not even sure it's possible to skip, but even having the ability to key the pin in with a controller would help.

**Comment by Fredrum** [comment]
Hmm that sounds a bit out of scope for me to be able to help with. Also still not sure what pin you are referring to. A screenshot might help?

**Comment by TechnoFlawgic** [comment]
I think this is more of an upstream problem, but it's just not something that seems to be taken into consideration since most platforms Chiaki runs on offer some form of keyboard input. I've set a login pin on my PS5 so that I don't have to worry about my kids trying to circumvent play time locks and other settings, but that means that whenever I start a session either locally or via remote play, I have to provide that pin. 

I'm not sure how to take screenshots on my Pi, but here's a screenshot from the Windows and Android clients, and one from the official Sony PS Remote Play app for a comparison as to what I'm trying to illustrate. 
![image](https://user-images.githubusercontent.com/4042630/152654294-6eb75643-ebf7-409e-ac54-db4ad0f70dc4.png)
![Screenshot_20220205-133759](https://user-images.githubusercontent.com/4042630/152654579-17dbfe7b-fede-4e91-9103-d68f29cb9ef0.png)
![image](https://user-images.githubusercontent.com/4042630/152654315-742430cd-0a8b-49f4-9fa7-12c048bdd070.png)

Basically, I'm trying to either pre-fill and send what that prompt is asking for via a config file or environment variable, or have some way to input via a controller so I don't have to keep a keyboard plugged in as this is running on a Retropie image.

**Comment by Fredrum** [comment]
Ok cool that makes it a bit clearer. But as you say that's a pretty off-piste problem and I would have to do something specific to it.
As I don't have that much time to spend on this project and there are other higher prio items that has to come first I wouldn't wait for a solution from me.

**Comment by TechnoFlawgic** [comment]
No worries, I already assumed it wasn't a huge issue for people considering the lack of any issues mentioning it in the upstream repo. Thanks anyways!
